src/utils/paragon-reset.scss

```diff
@@ -1,3 +1,4 @@
+@import '~edx-bootstrap/sass/open-edx/theme';
```

Comment:
😬 

Seems like an issue all right. How can we test that/how can we remove that dependency?

Reply:
[`!default` is kind of the opposite of `!important`](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#Variable_Defaults___default). Any variable definition in the same scope without `!default` will override the default -- even if that variable is defined *before* the default. So as long as themes define these vars without `!default`, we should be fine.

Reply:
Because we had concerns about this, I tested it out to make sure. Say we have a theme:

```scss
// ./src/_theme.scss
$red: #d9534f;
$brand-primary: $red;
```

As long as we import it before `@import "paragon-reset";`, then that theme will override anything in the openedx theme. The webpack config in our project for doing that:

```javascript
{
  loader: 'sass-loader',
    options: {
      sourceMap: true,
      data: '@import "red-theme"; @import "paragon-reset";',
      includePaths: [
        path.join(__dirname, '../src'),
        path.join(__dirname, '../node_modules/paragon/src/utils'),
        path.join(__dirname, '../node_modules'),
      ],
   },
},

```